apps/aecore/lib/aecore/governance/governance_constants.ex

```diff
@@ -101,5 +128,22 @@ defmodule Aecore.Governance.GovernanceConstants do
   @spec max_txs_per_block :: non_neg_integer()
   def max_txs_per_block, do: @max_txs_per_block
 
+  @spec block_gas_limit :: non_neg_integer()
   def block_gas_limit, do: @block_gas_limit
+
+  @spec default_tx_gas_price :: non_neg_integer()
+  def default_tx_gas_price, do: @default_tx_gas_price
+
+  @spec tx_base_gas(atom()) :: non_neg_integer()
+  def tx_base_gas(type) when type in @allowed_txs_types do
```

Comment:
I think this makes more sense, where the fee calculation was previously in code. Its not a constant.